src/test/test_dash.h

```diff
@@ -33,6 +40,7 @@ extern FastRandomContext g_insecure_rand_ctx;
  */
 extern bool g_mock_deterministic_tests;
 
+
```

Comment:
drop this newline

Reply:
Hello @PastaPastaPasta  dropped new line